Rock.Migrations/Migrations/201704281533422_WorkflowWebhookDefinedType.cs

```diff
@@ -0,0 +1,51 @@
+// <copyright>
+// Copyright by the Spark Development Network
+//
+// Licensed under the Rock Community License (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.rockrms.com/license
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+//
+namespace Rock.Migrations
+{
+    using System;
+    using System.Data.Entity.Migrations;
+
+    /// <summary>
+    ///
+    /// </summary>
+    public partial class WorkflowWebhookDefinedType : Rock.Migrations.RockMigration
+    {
+        /// <summary>
+        /// Operations to be performed during the upgrade process.
+        /// </summary>
+        public override void Up()
+        {
+            RockMigrationHelper.AddDefinedType( "Global", "Workflow Webhook", "Webhook to activate the Workflow", SystemGuid.DefinedType.WEBHOOK_TO_WORKFLOW );
+            RockMigrationHelper.AddDefinedTypeAttribute( SystemGuid.DefinedType.WEBHOOK_TO_WORKFLOW, "c0d0d7e2-c3b0-4004-abea-4bbfad10d5d2", "Url", "Url", "Request Url to activate the Workflow", 1, "", "5d9ac172-0d2f-495e-b435-de1a010acdbf" );
+            RockMigrationHelper.AddDefinedTypeAttribute( SystemGuid.DefinedType.WEBHOOK_TO_WORKFLOW, "9c204cd0-1233-41c5-818a-c5da439445aa", "Method", "Method", "Http Method Type allowed for the request", 2, "", "2d1693db-52aa-453c-b111-ebe9ca225317" );
```

Comment:
Can you use SystemGuids for the field types as well? For example instead of "c0d0d7e2-c3b0-4004-abea-4bbfad10d5d2", use SystemGuid.FieldType.URL_LINK. Same thing for all the attributes.